README.md

```diff
@@ -27,7 +27,10 @@ Yes, this is a YARP plugin
 ~~~
 If this is not the case, there could be some problems in finding the plugin. In that case, just move yourself to the `${CMAKE_INSTALL_PREFIX}/share/yarp` directory and launch the device from there.
 
-##### Device use 
+## Device use 
+
+### Via `inertial` network device (Deprecated)
```

Comment:
As this is deprecated, can you move it at the bottom? Thanks!

Reply:
Done :)